torch/utils/cpp_extension.py

```diff
@@ -168,16 +177,22 @@ def spawn(cmd):
                 # Using regex to match src, obj and include files
 
                 src_regex = re.compile('/T(p|c)(.*)')
-                src_list = [m.group(2) for m in (
-                    src_regex.match(elem) for elem in cmd) if m]
+                src_list = [
+                    m.group(2) for m in (src_regex.match(elem) for elem in cmd)
+                    if m
```

Comment:
lol really? we accept lines up to 120 chars, and that really looks silly

Reply:
Hmm I let Nuclide do this, I need to check if it uses our formatting guidelines